### PR TITLE
EXSWHTEC-118 - Implement tests for hipFuncGetAttribute

### PR DESCRIPTION
**Pull request by mirza-halilcevic**
- Negative parameter tests
- Basic positive test.

Depends-on: #21

### CODE REVIEW

File: catch/unit/module/hipFuncGetAttribute.cc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    HIP_CHECK_ERROR(hipFuncGetAttribute(nullptr, HIP_FUNC_ATTRIBUTE_BINARY_VERSION, kernel),
+                    hipErrorInvalidValue);
+  }
+
+  SECTION("invalid attribute") {
+    HIP_CHECK_ERROR(hipFuncGetAttribute(&value, static_cast<hipFunction_attribute>(-1), kernel),
+                    hipErrorInvalidValue);
+  }
+
+#if HT_NVIDIA // Disabled on AMD due to defect - EXSWHTEC-245
```
Comment (chrispaquot) [inline catch/unit/module/hipFuncGetAttribute.cc]: put in json and remove #if

File: catch/unit/module/hipModuleGetFunction.cc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+TEST_CASE("Unit_hipModuleGetFunction_Negative_Parameters") {
+  hipFunction_t kernel = nullptr;
+
+  SECTION("function == nullptr") {
+    HIP_CHECK_ERROR(hipModuleGetFunction(nullptr, GetModule(), "GlobalKernel"),
+                    hipErrorInvalidValue);
+  }
+
+// Disabled on AMD due to defect - EXSWHTEC-154
+#if HT_NVIDIA
```
Comment (chrispaquot) [inline catch/unit/module/hipModuleGetFunction.cc:48]: same and for the rest

**Review by chrispaquot — APPROVED** [review]

**Comment by rorake** [comment]
@mirza-halilcevic 
Will you please resolve the merge conflicts for this PR?

**Comment by rakesroy** [comment]
PR has been merged into develop branch via commit 34da562ff508f71424e158be131ea69aed35c80d.

**Pull request closed without merging**